### ISSUE TITLE
Auto-scale the text (done) but bug / race condition

### ISSUE THREAD

**Issue by jazzyjackson** [eraeco/pay#18]
Long endorsements currently wrap, and users may enter new lines, but ideally the endorsement fits on one line and its a module we can add to add an 'input' listener to scale the text until it fills the width of its container.

**Comment by amark** [comment]
solved.

THO NOTE:

endorsement field doesn't seem to be working when I tried it.

**Comment by amark** [comment]
auto-scaling/reflowing text is implemented, so I was gonna close, but then I noticed that sometimes it doesn't work prob because of my jQuery async loading of features & GUN loading of data is non-deterministic, so I need someone to review that events are probably fired and checked for and activated... I think we switched the code to trigger `onchange` logic so everything flows thru event system, this should be being triggered after GUN loads data and also on input changes. The one time I just saw it fail was on `payee` on a incognito tab fresh load of a check. Can anyone else also replicate? @Pineapple-1
